Python/chromedino_script/README.md

```diff
@@ -0,0 +1,26 @@
+## Chrome Dino Game Automation Using Python
+
+For doing automation I had used python libraries such as pyautogui and pillow .
+
+To play dino game we need to take care of two obstacles : first the cactus coming in between and second the bird flying above.
+
+To know the location of the obstacles first we need to find the X and Y locations of them on screen . Check coordinates.py file

```

Comment:
Can you make this more relevant? 

Usually follow this pattern while documenting your Code: 

- What does your Script does? 
- The files in them and their individual roles (Don't ask the user to check coordinates.py file) 
- Requirements to run these Scripts (Place them in a `requirements.txt` file